x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/handlers.ts

```diff
@@ -54,6 +66,31 @@ const exceptionListClientFromContext = (
   return exceptionLists;
 };
 
+const packagePolicyClientFromEndpointContext = (
+  endpointAppContext: EndpointAppContext
+): PackagePolicyServiceInterface => {
+  const packagePolicy = endpointAppContext.service.getPackagePolicyService();
+
+  if (!packagePolicy) {
+    throw new Error('Package policy service not found');
+  }
+
+  return packagePolicy;
+};
+
+const savedObjectClientFromEndpointContext = (
```

Comment:
You might not need this. you can get a scoped saved objects client right from the Route handler's `context` as:

```typescript
context.core.savedObjects.client
```